src/controllers/renewal-login.js

```diff
@@ -2,7 +2,7 @@ import jwt from 'jsonwebtoken';
 import jwkToPem from 'jwk-to-pem';
 import config from '../config.js';
 import axios from '../http-request.js';
-import {ReturnState} from './_base.js';
+import {monthsFromNow, yearsAgo} from '../utils/date-utils.js';
 
 /**
  * Get the TR-API's public key.
```

Comment:
Comments need to be updated in line with your code

Reply:
lines 23 - 42 of src/controllers/renewal-login.js

Reply:
I'm more in favour of removing them than deleting them, the code is self explanatory and comments add a useless overhead